ftl-project.toml

```diff
@@ -18,6 +18,9 @@ disable-ide-integration = true
   [modules.mysql]
     [modules.mysql.secrets]
       FTL_DSN_MYSQL_TESTDB = "inline://InJvb3Q6c2VjcmV0QHRjcCgxMjcuMC4wLjE6MTMzMDYpL215c3FsX3Rlc3RkYj9hbGxvd05hdGl2ZVBhc3N3b3Jkcz1UcnVlIg"
+  [modules.postgres]
```

Comment:
I'm not sure why this keeps getting created. It seems like something is pushing the provisioned DSNs into the secret manager.

Reply:
https://github.com/TBD54566975/ftl/pull/3501 should fix it, probably, for now can you remove it from the PR?

Reply:
oh whoops yeh